Minecraft pocket edition

```diff
@@ -0,0 +1 @@
+
```

Comment:
[Hello-World-master.zip](https://github.com/TheMinecraftModder/The-American-Flag-LuckyBlock-Mod/files/344542/Hello-World-master.zip)